src/client/components/Navigation/NavAssessment/Section/SectionItemLink/SectionItemLink.tsx

```diff
@@ -60,7 +59,7 @@ const SectionItemLink: React.FC<Props> = (props) => {
       <div className="nav-section__label">{Labels.getCycleLabel({ cycle, labels: subSection.props.labels, t })}</div>
       {!isDataExport && (
         <div className="nav-section__status-content">
-          <ReviewStatusMarker status={reviewStatus} />
+          <ReviewSummaryIndicator status={reviewStatus} />
```

Comment:
can the reviewSummaryIndicator have as prop only id and the hook useSectionReviewSummary (that should be renamed to useReviewSummary) be called directly in the component reviewSummaryIndicator?

Reply:
yes, but we have to differentiate if is an odp or not. maybe with a second parameter?

`const reviewStatus = useSectionReviewSummary(section.id)`
`const reviewStatus = useSectionReviewSummary(id)`
`const reviewStatus = useOdpReviewSummary(odpId)`